### PR TITLE
[PHAC] itap-piem.alpha-canada.ca

### PR DESCRIPTION
**Pull request by lucbelliveau**
# Summary | Résumé

new DNS entry for itap-piem.alpha-canada.ca

### CODE REVIEW

**Comment by CalvinRodo** [comment]
Will run PR and merge on Tuesday sorry for the delay.

File: terraform/itap-piem.alpha-canada.ca.tf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+resource "aws_route53_record" "itap-piem.alpha-canada.ca-A" {
```
Comment (CalvinRodo) [inline terraform/itap-piem.alpha-canada.ca.tf:1]: Terraform resources can't have `.` in the name please replace with underscores or dashes or remove all together

**Review by CalvinRodo — CHANGES_REQUESTED** [review]
See Comment

File: terraform/itap-piem.alpha-canada.ca.tf (hunk 1 above)
Comment (lucbelliveau) [reply terraform/itap-piem.alpha-canada.ca.tf]: My bad, updated.

File: terraform/itap-piem.alpha-canada.ca.tf [hunk 2]
```diff
@@ -0,0 +1,9 @@
+resource "aws_route53_record" "itap-piem-alpha-canada.ca-A" {
```
Comment (CalvinRodo) [inline terraform/itap-piem.alpha-canada.ca.tf:1]: Left one in there.

**Review by CalvinRodo — CHANGES_REQUESTED** [review]
left one period in the name

File: terraform/itap-piem.alpha-canada.ca.tf (hunk 2 above)
Comment (lucbelliveau) [reply terraform/itap-piem.alpha-canada.ca.tf]: Man I even had 2 coffees this morning.  Hopefully this will do it

**Comment by lucbelliveau** [comment]
@CalvinRodo I see the job failed but the error doesn't appear to be related to what I wrote (this time) - is there anything I need to do to push this through?

**Comment by CalvinRodo** [comment]
Nope as mentioned I'll have to recreate and run the p myself,, we don't allow repo secrets to be exposed to forked PRs for security reasons.

We have an item on our backlog to change how we authenticate with AWS so it doesn't require secrets until that time this is what we have to do.

**Comment by lucbelliveau** [comment]
Ok thanks for the update.

**Comment by lucbelliveau** [comment]
Thanks!

**Pull request closed without merging**